src/pages/test-report/cat-a-mod1/test-report.cat-a-mod1.page.ts

```diff
@@ -246,4 +250,41 @@ export class TestReportCatAMod1Page extends BasePageComponent {
   onTerminate = (): void => {
     this.modal.dismiss().then(() => this.navController.push(CAT_A_MOD1.DEBRIEF_PAGE));
   }
+
+  passTest = (): void => {
+    this.store$.dispatch(new AddDrivingFault({
+      competency: Competencies.clearance,
+      newFaultCount: 3,
+    }));
+    this.store$.dispatch(new AddDrivingFault({
+      competency: Competencies.followingDistance,
+      newFaultCount: 1,
+    }));
+    this.store$.dispatch(new AddDrivingFault({
+      competency: Competencies.moveOffSafety,
+      newFaultCount: 2,
+    }));
+    this.store$.dispatch(new SetActivityCode('1'));
```

Comment:
I think we're missing a call to calculate the result?

Reply:
See https://github.com/dvsa/mes-mobile-app/pull/901/

Reply:
I didn't see it when I looked at this example:
https://github.com/dvsa/mes-mobile-app/pull/1068/files

Is it needed?